api/src/main/resources/public/api/v1/metrics.raml

```diff
@@ -15,7 +15,16 @@ get:
               },
               "gauges": {
                 "name.of.gauge": {
-                  "value": 7248
+                  "value": 7248,
```

Comment:
Feels like Metronome should adapt the sliding window was well once it's merged.

Reply:
it uses the same metrics as Marathon, same classes, same serialization logic - same broken stuff, unfortunately :). So I am not sure what do you mean...